web-server/src/routes/v1/auth.ts

```diff
@@ -9,5 +9,6 @@ const router = Router();
 router.post('/login', [validatorLogin], login);
 router.post('/register', [validatorRegister], register);
 router.post('/change-password', [checkJwt, validatorChangePassword], changePassword);
+router.post('/change', changeGeneral);
```

Comment:
Yeah. `router.put('v1/users/update/:id')` is kind of where I'd expect this to be since we're updating a user and that has nothing to do with "authentication" (logging in)